package.json

```
@@ -1,17 +1,22 @@
 {
   "name": "jscad-text",
-  "version": "3.0.1",
+  "version": "3.0.2",
   "description": "JSCAD module to convert text to 2D paths, using TTF (and other) fonts",
-  "main": "dist/src/index.js",
-  "types": "dist/src/index.d.ts",
+  "type": "commonjs",
+  "main": "./dist/src/index.js",
+  "exports": "./dist/src/index.js",
+  "browser": "./dist/jscad-text.min.js",
+  "types": "./dist/src/index.d.ts",
```

I think some of these options are older style of exports. I don't think it's wrong as-is, but I think a more modern version might look like:

```suggestion
  "type": "commonjs",
  "main": "./dist/src/index.js", // legacy node require()
  "browser": "./dist/jscad-text.min.js", // legacy bundlers
  "types": "./dist/src/index.d.ts",
  "exports": {
    ".": {
      "types":   "./dist/src/index.d.ts",
      "require": "./dist/src/index.js", // node + bundlers (CJS)
      "browser": "./dist/jscad-text.min.js",  // umd <script> tag use
      "default": "./dist/src/index.js" // fallback
    }
  },
```